packages/database/src/migrations/20200924035244-DeleteOrphanMapOverlayRelations-modifies-data.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
Some orphan relations that I've forgot to deleted in my migrations a while ago that I found out while working on this task